diff --git a/node_modules/strapi-plugin-menus/server/content-types/menu-item.js b/node_modules/strapi-plugin-menus/server/content-types/menu-item.js
index c384c7a..04cea79 100644
--- a/node_modules/strapi-plugin-menus/server/content-types/menu-item.js
+++ b/node_modules/strapi-plugin-menus/server/content-types/menu-item.js
@@ -14,10 +14,10 @@ module.exports = {
   },
   pluginOptions: {
     'content-manager': {
-      visible: false,
+      visible: true,
     },
     'content-type-builder': {
-      visible: false,
+      visible: true,
     },
   },
   attributes: {
diff --git a/node_modules/strapi-plugin-menus/server/content-types/menu.js b/node_modules/strapi-plugin-menus/server/content-types/menu.js
index cabd02f..1cc1480 100644
--- a/node_modules/strapi-plugin-menus/server/content-types/menu.js
+++ b/node_modules/strapi-plugin-menus/server/content-types/menu.js
@@ -14,10 +14,10 @@ module.exports = {
   },
   pluginOptions: {
     'content-manager': {
-      visible: false,
+      visible: true,
     },
     'content-type-builder': {
-      visible: false,
+      visible: true,
     },
   },
   attributes: {
